libkmod/libkmod-module.c

```diff
@@ -1798,29 +1799,30 @@ static struct kmod_list *kmod_module_info_append(struct kmod_list **list, const
 
 static char *kmod_module_hex_to_str(const char *hex, size_t len)
 {
-	char *str;
-	int i;
-	int j;
+	static const char digits[] = "0123456789ABCDEF";
+	struct strbuf sbuf;
 	const size_t line_limit = 20;
-	size_t str_len;
+	size_t i;
```

Comment:
Nit: we can move the index declaration in the for loop.

Reply:
Adjusted